types/relationship.js/dist/relationship-mode.min.d.ts

```diff
@@ -1,6 +1,4 @@
-export as namespace relationshipMode;
-export = relationshipMode;
+export as namespace relationshipModes;
```

Comment:
I don't think this is correct, or maybe project site wasn't yet updated?

> ReferenceError: Can't find variable: relationshipModes

(as or 1.2.5)

Reply:
ok, i will check later